test/test_local_db.js

```diff
@@ -14,6 +15,8 @@ var createRequestReplica = {"__dbperapp": "someRemoteApp", "__fhdb": "someReplic
   {"_id": 1, "ReplicaField1": "ReplicaField1Data"}
 ]};
 
+var LOCAL_URL = 'mongodb://localhost:27017/FH_LOCAL';
```

Comment:
This is the URL that we typically suggest using during local development - this PR will allow folks to use it again.